src/interfaces.ts

```diff
@@ -12,6 +12,8 @@ export interface IBlock {
     fee: BigNumber;
     total: BigNumber;
     amount: BigNumber;
+    unikMintRewards?: BigNumber;
+    foundationRewards?: BigNumber;
```

Comment:
Why only on attribute for sender and forger and one for foundation ? (a comment in core PR)